bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -185,33 +248,34 @@ def check(m: discord.Message) -> bool:
                 word = "You guys"
                 winners = []
                 points_copy = list(player_data.values()).copy()
+
                 for _ in range(no_of_winners):
                     index = points_copy.index(highest_points)
                     winners.append(list(player_data.keys())[index])
                     points_copy[index] = 0
-                winners_mention = None
-                for winner in winners:
-                    winners_mention += f"{winner.mention} "
 
+                winners_mention = " ".join(winner.mention for winner in winners)
             else:
                 word = "You"
                 author_index = list(player_data.values()).index(highest_points)
                 winner = list(player_data.keys())[author_index]
                 winners_mention = winner.mention
+
             await channel.send(
-                f"Congratz {winners_mention} :tada: "
-                f"{word} have won this quiz game with a grand total of {highest_points} points!!"
+                f"Congratulations {winners_mention} :tada: "
+                f"{word} have won this quiz game with a grand total of {highest_points} points!"
             )
 
-    @property
     def category_embed(self) -> discord.Embed:
         """Build an embed showing all available trivia categories."""
         embed = discord.Embed(colour=discord.Colour.blue())
         embed.title = "The available question categories are:"
+        embed.set_footer(text="If a category is not chosen, a random one will be selected.")
```

Comment:
I recognise there is currently only one category. However, technically speaking this is not true. If no category is given, it will default to the general category; no randomness is involved. Were you not planning on implementing that now?

Reply:
I don't have time to add other categories, maybe later I will.
This PR was not to add more categories, It was to fix a problem mentioned in the PR description.

Reply:
I am asking about implementing randomness not adding more categories.

Reply:
There is only 1 category available so cant implement random now.

Reply:
Initially, I had a plan to add categories for every season(Halloween, Christmas, valentines, easter, pride, etc) but I guess that has to wait.

Reply:
You can still implement the randomness now even if there is currently only one category. It'll just always pick the same category until more are added. However, if you don't want to, that's OK.

Reply:
oooh like that, I think its gonna be better, I will do it.